lib/modules/manager/kotlin-script/readme.md

```diff
@@ -19,9 +14,10 @@ println("Hello world!")
 // ...
 ```
 
-By default, Renovate scans files only with `.main.kts` extension and not `.kts`, to avoid ambiguity with Gradle config
-files that have `.gradle.kts` extension. As there are cases where just `.kts` extension or no extension is used,
-Renovate can be [configured](https://docs.renovatebot.com/configuration-options/) to scan also these:
+By default, Renovate only scans files with the `.main.kts` extension and not `.kts`.
+This way Renovate avoids ambiguity with Gradle config files that use the `.gradle.kts` extension.
+
+If you have Kotlin Script files with the `.kts` extension, you may use the `fileMatch` config option to let Renovate update the `.kts` files:
```

Comment:
I've dropped the text that said: _or no extension is used,_. Do we want to add a configuration example for the case where no extensions are used? We only have an example for the `.kts` case now...

Reply:
```suggestion
If you want to manage other Kotlin Script files, you may use the `fileMatch` config option to let Renovate update these files:
```
This way we include the arbitrary Kotlin script files.